examples/pytorch_simple.py

```diff
@@ -40,40 +40,24 @@
 N_TEST_EXAMPLES = BATCHSIZE * 10
 
 
-class Net(nn.Module):
-    # Constructor for trial network.
-    def __init__(self, trial):
-        super(Net, self).__init__()
-        self.layers = []
-        self.dropouts = []
-
-        # We optimize the number of layers, hidden untis in each layer and drouputs.
```

Comment:
This comment is still relevant to the new code, and I think it is helpful for novice users of Optuna.
So, how about adding this comment to L44?

Reply:
That sounds great!